derive/src/lib.rs

```diff
@@ -96,6 +96,25 @@ fn emit_debug_impl<'a>(
     })
 }
 
+fn generate_derive_set(
+    nested: &syn::punctuated::Punctuated<NestedMeta, syn::token::Comma>,
+    allow_alias: bool,
+    derive_set: &mut HashSet<String>,
+) -> bool {
+    let mut make_custom_debug_impl = false;
+    for nested_meta in nested.iter() {
+        if let NestedMeta::Meta(inner_meta) = nested_meta {
+            let inner_meta_str = format!("{}", quote::quote!(#inner_meta));
+            if inner_meta_str.contains("Debug") && !allow_alias {
```

Comment:
This would break a derive `some_crate::DeriveThatHasDebugInTheName`.

Can you instead have it only "capture" a `Debug` derive if its path is exactly one of these:

- `Debug`
- `fmt::Debug`
- `::core::fmt::Debug`
- `core::fmt::Debug`
- `::std::fmt::Debug`
- `std::fmt::Debug`

It's unlikely (albeit possible) that these would conflict with other derives